lib/turn.rb

```diff
@@ -0,0 +1,71 @@
+class Turn
+    attr_reader :player1, :player2, :spoils_of_war, :winner
```

Comment:
```suggestion
    attr_reader :player1, :player2, :spoils_of_war
```

We don't need or want a `winner` `attr_reader` because we are defining a method for determining a winner on line 20. Remember an `attr_reader` is a shortcut for creating a method that allows us to read the value of an instance variable. If we define methods by the same name more than once, it can cause conflicts.